src/controllers/selectDirector.controller.ts

```diff
@@ -38,10 +39,10 @@ export class SelectDirectorController extends BaseController {
   @httpGet('')
   public async get(): Promise<string> {
     const form: Optional<SelectDirectorFormModel> = this.getFormFromSession()
-
+    const officerType = this.session.getDissolutionSession(this.httpContext.request)!.officerType
```

Comment:
getFormFromSession also calls `this.session.getDissolutionSession` - please refactor this so that we only retrieve the session once.